src/main/java/fox/spiteful/avaritia/items/ItemArmorInfinity.java

```diff
@@ -326,6 +326,7 @@ public void updatePlayerAbilityStatus(LivingUpdateEvent event) {
                         boolean swimming = player.isInsideOfMaterial(Material.water) || player.isInWater();
                         if (player.onGround || flying || swimming) {
                             boolean sneaking = player.isSneaking();
+                            player.stepHeight = player.isSneaking() ? 0.5F : 1F;
```

Comment:
Since there already is a flag `sneaking`, reuse that one instead of checking again here.

Reply:
Okay, I tried to change it